Automattic/sensei
# Update question title when changing question type

Fixes #2975

#### Changes proposed in this Pull Request:

* Update question title when changing question type

#### Testing instructions:

1. Go to 'wp-admin'.
2. Go to `Questions → Add New`.
3. Change the `Question Type` select.
5. The box title should update to the new type.


#### Screenshot / Video

![rsz_add_question_2](https://user-images.githubusercontent.com/8598283/76680806-0b1b3b80-6612-11ea-824c-b7a694fe48d0.png)
